example/app.conf

```diff
@@ -1,5 +1,15 @@
 # Example application configuration
 
+otel_trace          on;
+otel_service_name   example_service:nginx;
+otel_trace_context  propagate;
+otel_exporter {
+    endpoint        otel-collector:4317;
+    interval        5s;
+    batch_size      512;
+    batch_count     4;
+}
+
```

Comment:
These don't have to be here. Some like `otel_trace` & `otel_trace_context` can be located elsewhere i.e. in the `location` block.